src/actions/editor.js

```diff
@@ -118,14 +119,37 @@ export function sendData(formValues, contentLanguages, user, updateExisting = fa
         if(formValues.sub_events) {
             recurring = _.keys(formValues.sub_events).length > 0
         }
+        // Format descriptions to HTML
+        const descriptionTexts = formValues.description
+        for (const lang in formValues.description) {
+          const desc = formValues.description[lang].replace(/\n\n/g, "</p><p>").replace(/\n/g, "<br/>")
+          if (desc.indexOf('<p>') === 0 && desc.substr(desc.length - 4) === '</p>') {
+              descriptionTexts[lang] = desc;
+          } else {
+              descriptionTexts[lang] = `<p>${desc}</p>`
+          }
+        }
+        // Check for 'palvelukeskuskortti' in audience
+        if (formValues) {
+            if (formValues.audience && includes(formValues.audience, "https://api.hel.fi/linkedevents-test/v1/keyword/helsinki:aflfbat76e/")) {
```

Comment:
The id `helsinki:aflfbat76e` is hardcoded in the backend (as it is a rather special case), so it is safe to use just that.